cves/kernel/CVE-2017-16913.yml

```diff
@@ -1,4 +1,8 @@
 CVE: CVE-2017-16913
+# Patchset: https://lore.kernel.org/stable/lsq.1518322802.943358572@decadent.org.uk/
+# Report: https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2017-16913
+# Wayback Machine Link to Secunia: https://web.archive.org/web/20200227134938/https://www.flexera.com/products/operations/software-vulnerability-research/secunia-research/advisories.html
+#Patch - Patchset https://lore.kernel.org/stable/48094d82a68e9bb52f4463d46a8b0aa259c15a1c.1518036867.git.shuahkh@osg.samsung.com/T/#mbc7a5c48ddf3061e56394939ffb6274540126149 
```

Comment:
Why have you added all of these? It may be relevant, but it is also pulled from the NVD page which is easy enough to find.

Reply:
Read previous comment - https://github.com/VulnerabilityHistoryProject/vulnerabilities/pull/209#discussion_r1390334647